clients/pkg/promtail/targets/syslog/syslogtarget.go

```diff
@@ -225,7 +225,7 @@ func (t *SyslogTarget) handleMessageRFC3164(connLabels labels.Labels, msg syslog
 }
 
 func (t *SyslogTarget) handleMessage(connLabels labels.Labels, msg syslog.Message) {
-	if t.config.IsRFC3164Message {
+	if t.config.IsRFC3164Message() {
```

Comment:
I think switching on the `config.SyslogFormat` would be cleaner

Reply:
Here possible, but it is using in few more places, and it leads to `==` and need to export the contant name, or hardcoded strings...

So, I've decided to use trivial method which can be refactored.

Anyway, here only two RFC for Syslog, and I doubt that someone will make 3rd soon enough.